taccsite_cms/remote_cms_auth.py

```diff
@@ -0,0 +1,93 @@
+import inspect
+from django.http import HttpResponse, HttpResponseRedirect
+from django.contrib.auth.backends import ModelBackend
+from django.contrib.auth import get_user_model
+from django.contrib import auth as auth
+import requests
+import logging
+from django.conf import settings
+UserModel = get_user_model()
+
+logger = logging.getLogger(__name__)
+
+
+def verify_and_auth(request):
+    user = auth.authenticate(request)
```

Comment:
Confirming - this will not throw an exception when it fails to authenticate?

Reply:
Should not unless there is some non-standard exception. Django authors handle the most likely exceptions, (also the ones that provide the most flexibility), you can see the error handling here, https://github.com/django/django/blob/2.2.16/django/contrib/auth/__init__.py#L66